app/services/articles/builder.rb

```diff
@@ -7,7 +7,7 @@ def initialize(user, tag, prefill)
       @tag = tag
       @prefill = prefill
 
-      @editor_version2 = @user&.editor_version == "v2"
+      @editor_version2 = (@user&.setting && @user.setting.editor_version == "v2")
```

Comment:
We are mixing two styles here, `&.` and `&&`. 

```suggestion
      @editor_version2 = @user&.setting&.editor_version == "v2"
```

Reply:
😅 I learned about chaining "try's" like this I think exactly 2 days ago lol